rpm/BUILD.bazel

```diff
@@ -159,6 +159,76 @@ rpmtree(
     visibility = ["//visibility:public"],
 )
 
+rpmtree(
+    name = "generic-sidecar",
```

Comment:
Probably rename it to `sidecar-shim_arm64`  and `sidecar-shim_x86_64`

Reply:
I chose `aarch64` because that's how it is in the rest of this file.